Jenkinsfile

```
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+
+    stages {
+        stage('Checkout') {
+            steps {
+                checkout scm
+            }
+        }
+        stage('Dotnet Build') {
+            steps {
+                sh "dotnet build DotnetTemplate.sln"
+            }
+        }
+        stage('Dotnet Tests') {
+            steps {
+                sh "dotnet test DotnetTemplate.Web.Tests"
+            }
+        }
+        stage('Typescript Build') {
+            steps {
+                dir("DotnetTemplate.Web/") {
+                    sh "npm ci"
+                    sh "npm run build"
+                }
+            }
+        }
+        stage('Typescript Tests') {
+            steps {
+                dir ("DotnetTemplate.Web/") {
+                    sh "npm test"
+                }
+            }
+        }
+        stage('Typescript Lint') {
+            steps {
+                dir ("DotnetTemplate.Web/") {
+                    sh "npm run lint"
+                }
+            }
+        }
+    }
+
+    post {
+        failure {
+            slack(":red_circle: Build failed", "danger")
+        }
+        success {
+            slack(":excellent: Build succeeded", "good")
```

Unfortunately `:excellent:` isn't included in Slack by default.
```suggestion
            slack(":heavy_check_mark: Build succeeded", "good")
```